src/github/push.ts

```diff
@@ -35,8 +35,8 @@ export default async (context: Context, jiraClient): Promise<void> => {
 		return;
 	}
 
-	// If there's less than 20 commits (the number of commits the github API returns per call), just process it immediately
-	if(payload.commits?.length < 20 && await booleanFlag(BooleanFlags.PROCESS_PUSHES_IMMEDIATELY, true, jiraClient.baseURL)) {
+	// If there's less than 100 commits (the max number of commits the github API returns per call), just process it immediately
```

Comment:
This comment seems off. Why should the number of commits the GitHub API returns be relevant in this case? The commits are in the payload of the webhook, so we don't need to call the GitHub API?

Reply:
graphql can return 100 nodes, so if there's more than 100 commits, we need to add it to the queue to be processed (think force push or repository move).

I figured it's easier to just use the code we already have to get all commits from said push because the graphql query has all the information we need, while the webhooks is missing several bits of data (like author), that we would need to call the API either way.